modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java

```diff
@@ -216,6 +229,9 @@ private JSONObject _getJSONObject(
 				object)
 		).put(
 			"canonicalURL", canonicalURL
+		).put(
+			"detailsReferralEnabled",
+			_ffAnalyticsReportsDetailsConfiguration.referralEnabled()
```

Comment:
Please, could we do it Frontend aseptic? 

I think @dgarciasarai suggested that she would paint the link only if the data were filled, right?